src/import/importd.c

```diff
@@ -1351,6 +1359,28 @@ static int manager_run(Manager *m) {
                         m);
 }
 
+static void parse_env(Manager *m) {
+        int r;
+
+        assert(m);
+
+        /* Same as src/import/{import,pull}.c:
+         * Let's make these relatively low-level settings also controllable via env vars. User can then set
+         * them for systemd-importd.service if they like to tweak behaviour */
+
+        r = getenv_bool("SYSTEMD_IMPORT_BTRFS_SUBVOL");
+        if (r >= 0)
+                m->use_btrfs_subvol = (r > 0);
```

Comment:
just write:

```c
m->use_btrfs_subvol = r;
```

after all, you ensured the field is not negative, and thus we can safely rely on the implicit int → bool conversion here, after all getenv_bool() explicitly returns a boolean-like value already.

Reply:
Indeed, fixed both.